app/uk/gov/hmrc/thirdpartydeveloperfrontend/domain/models/emailpreferences/emailpreferences.scala

```diff
@@ -38,29 +36,46 @@ object EmailPreferences {
   def noPreferences: EmailPreferences = EmailPreferences(List.empty, Set.empty)
 }
 
-sealed abstract class EmailTopic(val value: String, val displayName: String, val description: String, val displayOrder: Byte) extends StringEnumEntry
```

Comment:
Did value really go away?

Reply:
Also not surer that 4 defs in companion might be better than 4 fields

Reply:
Ah - Value is the toString equivalent.

Reply:
We tested this manually with local testing, connecting to the backend. Seemed to work fine.

Reply:
Yeah - ignore the value thing - further down I saw how it was being used hence comment 3 of mine.